templates/commands/goldentest/test_funcs.go

```diff
@@ -156,10 +156,6 @@ func renderTestCases(ctx context.Context, testCases []*TestCase, location string
 func renderTestCase(ctx context.Context, templateDir, outputDir string, tc *TestCase) error {
 	testDir := filepath.Join(outputDir, goldenTestDir, tc.TestName, testDataDir)
 
-	if err := os.RemoveAll(testDir); err != nil {
```

Comment:
Is this the wrong logic that we want to remove?

Reply:
This was supposed to be deleted in a previous PR but seems to have been lost in a merge conflict, so I'm deleting it again